server/dummyControllers/requestController.js

```diff
@@ -36,6 +37,44 @@ class RequestController {
       });
     }
   }
+
+  /**
+ * it GET all requests
+ * @param {string} req
+ * @param {string} res
+ * @returns {object} object
+ */
+  static getARequest(req, res) {
+    const id = parseInt(req.params.id, 10);
+
+    // get a request
+    const newRequest = requests.filter(request => request.id === id);
+
+    // if request is not found
+    if (newRequest.length === 0) {
+      return res.status(404).json({
+        status: 'error',
+        message: 'request not found'
+      });
+    }
+    // get user in users db
+    const foundUser = users.some(user => user.name === newRequest[0].name);
```

Comment:
This function returns a request I am wondering why you are returning users in the same function. Always abide by the SRP principle

Reply:
ok